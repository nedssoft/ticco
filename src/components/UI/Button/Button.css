.Button {
  background-color: transparent;
  border: none;
  color: white;
  outline: none;
  cursor: pointer;
  font: inherit;
  padding: 10px;
  margin: 10px;
  font-weight: bold;
}

.Button:first-of-type {
  margin-left: 0;
  padding-left: 0;
}

.Success {
  color: #5C9210;
}

.Danger {
  color: #944317;
}
.Button:disabled {
  color: #ccc;
  cursor: not-allowed;
}
.btnDefault{
  padding: 10px 0 10px 0;
  text-align: center;
  line-height: 20px;
  background-color: #EE6860;
  letter-spacing: 0.3px;
  color: aliceblue;
  border: none;
  outline: none;
  font-size: 1em;
  font-weight: 600;
  text-decoration: none;
  cursor: pointer; 
}