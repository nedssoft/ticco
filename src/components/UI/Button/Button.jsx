import React from 'react'
import classes from './Button.css'

const Button = ({ clicked, children, btnType, disabled}) => {
  return (
    <button 
      onClick={clicked}
      disabled={disabled}
      className={[classes.Button, classes[btnType]].join(' ')}
    >
      { children }
    </button>
      
  )
}

export default Button
