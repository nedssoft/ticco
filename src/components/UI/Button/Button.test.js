import React from 'react';
import { shallow } from 'enzyme'
import Button from './Button'

describe('Auth Components', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Button btnType='Submit'>Submit</Button>)
  })
 it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
 })
 
 it('should render Button component', () => {
  expect(wrapper.find('button')).toHaveLength(1)
 })
 
})