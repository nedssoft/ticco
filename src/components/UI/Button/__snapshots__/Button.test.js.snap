// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Components should render correctly 1`] = `
<button
  className="Button Submit"
>
  Submit
</button>
`;
